src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -256,6 +256,8 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
         .value
     },
     publishMicrosite := Def.task {
+      // I don't know how to fix this for sbt 1.0.
```

Comment:
Maybe you can use as we did here:

https://github.com/47deg/sbt-org-policies/pull/510/files#diff-09ec74f5913120bc52ff45882b201224R88